src/lib/API/api.js

```diff
@@ -505,7 +510,10 @@ export class APIService {
     const txs = []
     const explorer = Config.ethereum[chainId].explorerAPI
 
-    const sender32 = `0x${sender.toLowerCase().slice(2).padStart(64, '0')}`
+    const sender32 = `0x${sender
+      .toLowerCase()
```

Comment:
there should be utils at web3 like normalise address, padValue/padBytes etc

Reply:
so I have to create an instance of web3 in the API class, just to add padLeft?
not sure I understand what this solves?

Reply:
import { padLeft } from 'web3-utils'

Reply:
done